styles.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: content-box;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.menu {
+  width: 18px;
+  height: 20px;
+}
+
+.top-menu {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 25px;
+  width: 375px;
+  height: 12px;
+}
+
+.toolbarlogo {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  color: #fff;
+  background-color: #3c3a39;
+  height: 20px;
+}
```

Comment:
- [ ] Fix the distance between your headlines and toolbar here so it matches exactly the template
![image](https://user-images.githubusercontent.com/74744787/158875023-680fefef-2edf-4b3f-8721-abc9169d9a67.png)


![image](https://user-images.githubusercontent.com/74744787/158875092-8aaa5b86-f1ec-4a60-8c05-737b9d4ba2bb.png)